SwiftFlow/CoreTypes/Reducer.swift

```diff
@@ -25,3 +25,23 @@ extension Reducer {
     }
 
 }
+
+protocol PartialReducer: Reducer {
+    
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)